data/json/professions.json

```diff
@@ -4232,5 +4232,65 @@
     "points": -3,
     "items": { "both": [ "shorts", "heroin", "syringe" ], "male": [ "briefs" ], "female": [ "bra", "panties" ] },
     "addictions": [ { "intensity": 40, "type": "opiate" } ]
+  },
+  {
+    "type": "profession",
+    "ident": "yoga_instructor",
+    "name": "Yoga Instructor",
+    "description": "You were in the middle of downward dog. Now you need to get down and out.",
+    "points": 2,
+    "traits": [ "FAST_REFLEXES", "LIGHTEATER", "THINSKIN", "FLIMSY" ],
```

Comment:
I fail to see how any of the traits are justified outside of massive stereotyping.

Reply:
Following up from another reply. The only one here I agree with you on is Flimsy. Maybe Thinskin. I needed a negative to offset the bonus to dodge.

The average yoga instructor is flexible, dexterous, slim. Regular yoga is not very taxing on the body's muscle system and does not equate to a significant adaptation neurologically through induced loading. Power yoga on the other hand is a fork of yoga that is specifically designed to be load inducing utilizing the body's weight and static holds to induce a neurological adaptation period.

In short, flexibility training consumes a lot less calories than loading the muscle groups does.

Reply:
I will also try to supply screenshots. I am pooped right now. I will follow up.

Reply:
Pictures updated.